file_transfer_agent_test.go

```diff
@@ -968,7 +981,9 @@ func testUploadDownloadOneFile(t *testing.T, isStream bool) {
 	if isStream {
 		fileStream, _ := os.Open(uploadFile)
 		ctx := WithFileStream(context.Background(), fileStream)
-		uploadMeta.srcStream = getFileStream(ctx)
+		if uploadMeta.srcStream, err = getFileStream(ctx); err != nil {
+			assertNilF(t, err)
```

Comment:
We can skip the `if` and just do the assertion

Reply:
Right, done.